### PR TITLE
added automation-jira tag

### PR DESCRIPTION
**Pull request by jhellar**
# Description

Added `automation-jira-created` tag, so that it is easier to filter manual test cases for which automation jira does not exist yet. This could be improved by updating tooling to allow creating jiras for test cases that are not covered yet by jira ticket.

### CODE REVIEW

File: test-cases/tests/alerts/c03-verify-that-alerting-mechanism-works.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 estimate: 30m
 tags:
   - happy-path
+  - automation-jira-created
```
Comment (b1zzu) [inline test-cases/tests/alerts/c03-verify-that-alerting-mechanism-works.md:5]: @jhellar I think that if we have the `automation_jiras` we don't need also a tag, we can modify the tool to automatically add the tag if it finds an `automation_jiras`

Comment (jhellar) [reply test-cases/tests/alerts/c03-verify-that-alerting-mechanism-works.md]: ok, should be fixed now

**Comment by b1zzu** [comment]
@jhellar do we have some follow up tasks to create a new cmd in the tools to automatically create the tasks for automating the test cases?

**Comment by b1zzu** [comment]
@jhellar I forgot about this, but we should at least but a small note in the README.md

**Comment by jhellar** [comment]
@b1zzu I've created the task in https://issues.redhat.com/browse/INTLY-8430. Will send another PR for the readme.

**Comment by b1zzu** [comment]
Thanks @jhellar  :)

**Pull request merged**